app/controllers/PayPal.scala

```diff
@@ -53,15 +53,15 @@ class PayPal(
   def resultFromEmailOption(email: Option[Email]): Result = {
     val redirect = Redirect("/contribute/one-off/thankyou")
     email.fold(redirect)(e => {
-      logger.info("Redirecting to thank you page with email in flash session")
+      SafeLogger.info("Redirecting to thank you page with email in flash session")
       redirect.flashing("email" -> e.value)
     })
   }
 
   def execute(): Action[AnyContent] = PrivateAction.async { implicit request =>
     contributionsFrontendService.execute(request).fold(
       err => {
-        logger.error("Error making paypal payment: " + err.getMessage)
+        SafeLogger.error(scrub"Error making paypal payment: ${err.getCause}")
```

Comment:
to get proper stack traces in kibana I'd rather have a version of `error(...)` that still takes the throwable as well as a scrubbed string, and only passes the exception details to kibana and not sentry.  It could optionally send only the stack trace not the message to sentry to enable better duplicate detection (as a future extenson)